src/AddTodoForm.jsx

```diff
@@ -11,21 +11,20 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   const handleAddTodo = (event) => {
     event.preventDefault();
-    if (todoTitle.trim()) {
-      onAddTodo({ title: todoTitle, id: Date.now() });
-      setTodoTitle('');
-    }
+    if (todoTitle === '') return;
+    onAddTodo(todoTitle);
+    setTodoTitle('');
```

Comment:
Good use of validation for the todo length. You could improve that using the trim method in case the user inputs a "   " string (many blank spaces)